dts/bindings/sensor/nxp,kinetis-temperature.yaml

```diff
@@ -12,12 +12,10 @@ properties:
       required: true
 
     io-channels:
-      type: phandle-array
       required: true
       description: ADC channels for temperature sensor and bandgap voltage
```

Comment:
Why was the description removed for some bindings, but not others?

Reply:
I only left in the binding specific descriptions where I thought they convey additional information (as noted in the PR description and commit message). Would you like me to leave them all in?

Reply:
I would, please.  I think they help provide a clue to the underlying technology, e.g. NTC thermistor or photoresistor vice phototransistor.

Reply:
@pabigot, @mbolivar-nordic Thank, I appreciate your feedback. I have re-added the binding-specific descriptions.